src/Compilers/VisualBasic/Portable/Compilation/SyntaxTreeSemanticModel.vb

```diff
@@ -275,8 +275,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 Dim newSymbolInfo = GetSymbolInfoForNode(options, New BoundNodeSummary(bound, bound, Nothing), binderOpt:=Nothing)
 
-                If Not newSymbolInfo.GetAllSymbols().IsDefaultOrEmpty Then
-                    Return SymbolInfoFactory.Create(newSymbolInfo.GetAllSymbols(), LookupResultKind.NotATypeOrNamespace)
+                Dim allSymbols = newSymbolInfo.GetAllSymbols()
```

Comment:
avoids wasteful double-allocation.  i can revert if that extra allocation is desired.